web-registry/src/components/organisms/BasketOverview.tsx

```diff
@@ -0,0 +1,268 @@
+import { Label } from 'web-components/lib/components/label';
+import { parseText } from 'web-components/lib/utils/textParser';
+import { formatNumber } from 'web-components/lib/components/table';
+import { formatDate } from 'web-components/lib/utils/format';
+
+import { OptimizedImage } from '../atoms/OptimizedImage';
+import topoImg from '../../assets/background-contour-2.svg';
+import forestImg from '../../assets/forest-token.png';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  content: {
+    display: 'flex',
+    width: '100%',
+    padding: theme.spacing(0, 5),
+    [theme.breakpoints.up('sm')]: {
+      padding: theme.spacing(30, 5),
+      maxWidth: theme.breakpoints.values.lg,
+    },
+    [theme.breakpoints.down('lg')]: {
+      flexDirection: 'column',
+    },
+  },
+  imageContainer: {
+    position: 'relative',
+    width: '100%',
+    [theme.breakpoints.down('sm')]: {
+      height: theme.spacing(57.5),
+      overflowY: 'clip',
+    },
+  },
+  image: {
+    width: '100%',
+    maxWidth: theme.spacing(103),
+    [theme.breakpoints.down('sm')]: {
+      height: theme.spacing(76.4),
+      minWidth: theme.spacing(103),
+      overflow: 'hidden',
+      position: 'absolute',
+      top: '50%',
+      left: '50%',
+      transform: 'translate(-50%, -50%)',
+    },
+  },
+  textContainer: {
+    display: 'flex',
+    flex: 1,
+    flexDirection: 'column',
+    [theme.breakpoints.down('lg')]: {
+      paddingLeft: theme.spacing(8),
+    },
+    [theme.breakpoints.down('sm')]: {
+      width: '100%',
+      padding: theme.spacing(8, 0),
+      '&:last-child': {
+        paddingBottom: theme.spacing(15),
+      },
+    },
+  },
+  title: {
+    lineHeight: theme.spacing(9),
+    marginBottom: theme.spacing(2),
+    [theme.breakpoints.down('sm')]: {
+      marginBottom: theme.spacing(0),
+    },
+  },
+  basketDenom: {
+    fontWeight: 'bold',
+    marginTop: theme.spacing(2),
+    color: theme.palette.info.main,
+  },
+  basketDescription: {
+    fontSize: theme.typography.pxToRem(22),
+    marginTop: theme.spacing(2),
+    [theme.breakpoints.down('md')]: {
+      fontSize: theme.typography.pxToRem(18),
+    },
+  },
+  card: {
+    margin: theme.spacing(8, 0, 0),
+    padding: theme.spacing(3, 5, 7),
+  },
+}));
+
+export interface BasketOverviewProps {
+  name: string;
+  basketDenom: string;
+  description: string;
+  totalAmount: number;
+  curator: string;
+  allowedCreditClasses: string[];
+  minStartDate?: string;
+  startDateWindow?: string;
+}
+
+const BasketOverview: React.FC<BasketOverviewProps> = ({
+  name,
+  basketDenom,
+  description,
+  totalAmount,
+  curator,
+  allowedCreditClasses,
+  minStartDate,
+  startDateWindow,
+}) => {
+  const styles = useStyles();
+
+  return (
+    <Box
+      sx={theme => ({
+        display: 'flex',
+        justifyContent: 'center',
+        borderBottom: `1px solid ${theme.palette.grey[100]}`,
+        backgroundColor: theme.palette.grey[50],
```

Comment:
You might already know this, but the `theme` callback isn't necessary here, you can also write this as 
```suggestion
        backgroundColor: 'grey.50'
```
though it'd take splitting `borderBottom` into two properties...
```tsx
borderBottom: 1,
borderColor: 'grey.100'
```